entities/crew_entity.py

```diff
@@ -1,25 +1,156 @@
-import pygame
-import sys
-
-
-class Crew(object):
-    def __init__(self):
-        self.screen = None
-        self.clock = None
-
-    def init(self, screen_size=(800, 600)):
-        pygame.init()
-
-        self.screen = pygame.display.set_mode(screen_size)
-        self.clock = pygame.time.Clock()
-
-        pygame.display.set_caption('Crew')
-
-        running = True
-        while running:
-
-            for event in pygame.event.get():
-            
-                if event.type == pygame.QUIT:
-                    running = False
-                    sys.exit()
+import pygame
+from util.change_window_size_util import change_size_window
+
+
+class Crew(object):
+    def __init__(self):
+        self.screen = None
+        self.clock = None
+
+    def init(self):
+        pygame.init()
+
+        info = pygame.display.Info()  # Tem que ser antes do .set_mode()
+        self.screen_width, self.screen_height = info.current_w, info.current_h
```

Comment:
```suggestion
        self.screen_dimensions = (info.current_w, info.current_h)
```

Pode ser legal juntar essas variáveis numa tupla.